src/tests/ftest/launch.py

```diff
@@ -395,6 +406,14 @@ def set_provider_environment(interface, args):
     os.environ["CRT_PHY_ADDR_STR"] = provider
     print("Testing with CRT_PHY_ADDR_STR={}".format(os.environ["CRT_PHY_ADDR_STR"]))
 
+    # pylint: disable=import-outside-toplevel
+    sys.path.append(os.path.join(os.getcwd(), "util"))
+    from network_utils import get_common_provider
+    print("<>" * 50)
+    servers = NodeSet(args.test_servers)
+    print("get_common_provider: {}".format(get_common_provider(servers, interface)))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string